chronos/transports/stream.nim

```diff
@@ -639,6 +639,7 @@ when defined(windows):
                 child: StreamTransport = nil,
                 localAddress = TransportAddress(),
                 flags: set[SocketFlags] = {},
+                dualstack = DualStackType.Auto
```

Comment:
why do we need dualstack on connect?

Reply:
We need it, because otherwise IPv4 mapped address will not work when used in `connect()`.